rfc/0009-rancher-integration-charts.md

```diff
@@ -175,3 +179,9 @@ chart>/package.yaml` packages. This increases maintenance costs.
 While soft (optional) dependencies for Kubewarden charts are ok, they are
 implemented directly in Rancher Explorer UI logic, as per Rancher policy. This
 could be improved.
+
+All needed images for the charts need to be mirrored in the Rancher DockerHub
+repository, instead of being consumed from upstream.  For normal container
+images this is no problem, but not so for Wasm modules. We need to raise the
+request to provide a Rancher OCI registry, use Kubewarden's upstream one, or
```

Comment:
Please note this question. I suppose for backwards compatibility, we would end with a new Rancher OCI registry in addition to the Rancher Dockerhub registry.

Reply:
This question is of course also present in RFC 10, for policy charts.